weave/flow/eval_imperative.py

```diff
@@ -369,6 +378,10 @@ def ui_url(self) -> str | None:
             return None
         return self._evaluate_call.ui_url
 
+    @property
+    def attributes(self) -> dict[str, Any]:
+        return IMPERATIVE_EVAL_MARKER | self.eval_attributes
```

Comment:
will our marker get clobbered? shouldn't this be the other way around?

Reply:
good point